pages/common/mutagen.md

```diff
@@ -0,0 +1,28 @@
+# mutagen
+
+> Real-time file synchronization and network forwarding tool.
+> More information: <https://mutagen.io>.
+
+- Start a synchronization session between a local directory and a remote host:
+
+`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{user@host:/path/to/remote/directory/}}`
+
+- Start a synchronization session between a local directory and a Docker container:
+
+`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} {{docker://user@container_name/path/to/remote/directory/}}`
```

Comment:
```suggestion
`mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} docker://{{user}}@{{container_name/path/to/remote/directory/}}`
```

Reply:
Maybe separate the container name and the path then?
```bash
mutagen sync create --name={{session-name}} {{/path/to/local/directory/}} docker://{{user}}@{{container_name}}{{/path/to/remote/directory/}}
```